proto/metrics/metrics.proto

```diff
@@ -373,6 +373,21 @@ message appMetric {
   repeated appContainerMetric container = 7;
 
   AppMemoryMetric appMemory = 8;
+
+  repeated appPatchEnvelopeMetric patchEnvelope = 9;
```

Comment:
This is under the appMetric. Do we care about reporting any metrics for patch envelopes which are not (yet) associated with any app instances?

Reply:
Depends on weather we want sidecar container to be an app entity